response.go

```diff
@@ -286,6 +286,50 @@ func statusRangeText(code int) string {
 	}
 }
 
+// StatusList succeeds if response matches with any given status code list
+//
+// Example:
+//
+//	resp := NewResponse(t, response)
+//	resp.StatusList(http.StatusForbidden, http.StatusUnauthorized)
+func (r *Response) StatusList(values ...int) *Response {
```

Comment:
Nitpick: let's move this method before `func statusCodeText` and `func statusRangeText`, so that first we have public Status funcs, then we have private status helpers.